servo-tidy.toml

```diff
@@ -32,9 +32,14 @@ rand = [
 [ignore]
 # Ignored packages with duplicated versions
 packages = [
+  "base64",
```

Comment:
:sob:

Reply:
This is a question for @gw3583

Reply:
This sounds like it would add complexity to WR - e.g. the shader cache would now have to encode which type of hash it's using etc.